src/element/shape.js

```diff
@@ -237,7 +237,7 @@ export class ShapeElement extends Element {
     fill.enabled = true;
     fill.name    = data.nm;
     fill.opacity = ShapeElement.createOpacity(data.o);
-    this.fill    = fill;
+    this.fillRGBA    = fill;
```

Comment:
I have changed the property fill fillRGBA , for not override the new property for the class PIXI.Graphics fill which is readonly in the new version 5 of PIXI. [link](http://pixijs.download/dev/docs/PIXI.Graphics.html#fill)